tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I went with `built_spec/` instead because keeping the files in the same location relative to `index.js` is important.